### PR TITLE
Remove debug

### PR DESCRIPTION
**Pull request by ocefpaf**
Addresses https://github.com/conda-forge/libnetcdf-feedstock/pull/58#discussion_r236054451

Ping @isuruf

### CODE REVIEW

**Comment by ocefpaf** [comment]
@kmuehlbauer regarding the remote tests, I tested the 4.6.2 version and only 

```
ncdump -h "https://data.nodc.noaa.gov/thredds/dodsC/ioos/sccoos/scripps_pier/scripps_pier-2016.nc"
```

seems to be failing (but it passes with the previous version). However, none of them passes at build time, not sure if that is related to the curl version being pulled of the build env :-/

File: recipe/build.sh [hunk 1]
```diff
@@ -69,7 +68,6 @@ cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
       -DENABLE_DOXYGEN=OFF \
       -DENABLE_LOGGING=ON \
       -DCMAKE_C_FLAGS_RELEASE=${CFLAGS} \
```
Comment (isuruf) [inline recipe/build.sh]: This should be removed as well.

Comment (ocefpaf) [reply recipe/build.sh]: What is the reason for the removal of these two?

Comment (isuruf) [reply recipe/build.sh]: https://github.com/conda-forge/netcdf4-feedstock/issues/61
https://github.com/conda-forge/libnetcdf-feedstock/pull/56

**Comment by kmuehlbauer** [comment]
@ocefpaf We have this conda-build pin to `3.16.2` in `build.sh`. This is not connected to the failure we see? I'm struggling to get a grip here.

**Comment by ocefpaf** [comment]
> @ocefpaf We have this conda-build pin to `3.16.2` in `build.sh`.

We need to investigate this to avoid the workaround but as you notice we ate hitting another issue here :-(

> This is not connected to the failure we see? I'm struggling to get a grip here.

Same here, no clue... (I'm in a bad time with the day job, reports season! So I won't be able to follow up on this PR, sorry.)

**Comment by jschueller** [comment]
@conda-forge-admin, please rerender

**Comment by manishsoni0291** [comment]
Please tell me if somebody find a solution how to do it

**Comment by kmuehlbauer** [comment]
@manishsoni0291 This was resolved with https://github.com/conda-forge/libnetcdf-feedstock/pull/78

If this is still a problem please open a new issue with enough information to reproduce...

**Pull request closed without merging**